operator/internal/status/lokistack.go

```diff
@@ -113,18 +123,26 @@ func generateCondition(ctx context.Context, cs *lokiv1.LokiStackComponentStatus,
 	return conditionReady, nil
 }
 
-func checkForZoneawareNodes(ctx context.Context, k client.Client, zones []lokiv1.ZoneSpec) (bool, error) {
+func checkForZoneawareNodes(ctx context.Context, k client.Client, zones []lokiv1.ZoneSpec) (bool, bool, error) {
```

Comment:
You can give the return values names:

```suggestion
func checkForZoneawareNodes(ctx context.Context, k client.Client, zones []lokiv1.ZoneSpec) (nodesOk bool, labelsOk bool, err error) {
```